index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <nav class="menu">
+                <h4 class="logo">My Logo</h4>        
+                <img src="Icon - Menu.png" alt="Icon-menu"></nav>
+        </header>
+        
+    
+        <main class="about">
+            <section class="intro">
+                <h2>I’m Karthika. <br>
```

Comment:
- It is important  that this text which serves as your main title is  represented with an `h1` tag. If you want that google (and other search engines) to know about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. :+1: :sparkles: